todo_app/templates/index.html

```diff
@@ -12,7 +12,17 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+          <uL>
+            {% for item in items %}
+              <li>{{ item.id }} - {{ item.status }} - {{ item.title }}</li>
+            {% endfor %}
+          </uL>
       </ul>
     </div>
   </div>
+  <form action="/additem" method="POST">
+    <label for="nitem">New ToDo entry:</label><br>
+    <input type="text" id="nitem" name="nitem" value="Enter ToDo"><br><br>
```

Comment:
Instead of using `<br>` tags to introduce spacing, it's more correct to use styling. If you just want line breaks, then you can use tags with display: "block". E.g. `div` tags have that by default. 

To introduce spacing, we have the bootstrap styling applied already, so can try adding a class such as `class="mb-3"` to set a "margin-bottom". I think bootstrap accepts numbers from 1-5 for the size of the space.